clang/unittests/Format/FormatTest.cpp

```diff
@@ -9938,12 +9984,16 @@ TEST_F(FormatTest, ReturnTypeBreakingStyle) {
                "    return 1;\n"
                "  }\n"
                "  int g();\n"
+               "  long\n"
+               "  foooooooooooooooooooooooooooo::baaaaaaaaaaaaaaaaaaaar();\n"
```

Comment:
Delete it because it's not a definition?

Reply:
Deleted.